connector/MongoDBAtlasODBC.pq

```diff
@@ -111,7 +111,7 @@ shared GetConnectionString = (uri as text, database as text) as record =>
     let
         DatabaseNotSet = (database = ""),
         ConnectionString = [
-            DRIVER = "ADF_ODBC_DRIVER",
+            DRIVER = "MONGODB_ODBC_DRIVER",

```

Comment:
umh, this is what you'll see in ODBC administrator. Does not look much better. Let's ping Alexi again.

Reply:
Updated the name to `MongoDB Atlas SQL ODBC Driver`